cmd/migrations.go

```diff
@@ -0,0 +1,45 @@
+package cmd
+
+import (
+	"os"
+	postgres2 "parcel-service/internal/pkg/postgres"
+
+	_ "github.com/golang-migrate/migrate/v4/database/postgres"
+	_ "github.com/golang-migrate/migrate/v4/source/file"
```

Comment:
what is the purpose of file at here?

Reply:
This line `_"github.com/golang-migrate/migrate/v4/source/file"` is to import file driver in our project.